common/src/main/java/tachyon/worker/DataServerMessage.java

```diff
@@ -160,7 +160,7 @@ public static DataServerMessage createBlockResponseMessage(boolean toSend, long
 
   private long mLength;
 
-  private int mLockId = -1;
+  private long mLockId = -1L;
```

Comment:
This `mLockId` field is not really used in RPCMessage sent over wire. Add a TODO to remove this field from this class when `NIODataServer` is deprecated.